doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -280,7 +280,24 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
-.. rst-class:: v2.4.1 v2-4-0 v2-3-0 v2-2-0
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0
+
+KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test for lock sample.
+  The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0 in lock sample.
+
+  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in samples/matter/lock/src/lock.zap by ZAP Tool and regenerate files.
+
+
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
+
+KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup.
+  The lack of response can happen because of the Thread interface being unnecessarily reset.
+  This can result in the TC-CNET-4.10 Matter Certification test failing.
```

Comment:
```suggestion
  This can happen because of the Thread interface being unnecessarily reset and can result in the TC-CNET-4.10 Matter Certification test failing.
```
I think this reads just a little bit better. Sorry for going back and forth.